source/js/buyers-guide/bg-main.js

```diff
@@ -194,6 +223,31 @@ let main = {
         injectDonateModal(donationModal, modalOptions);
       }
     */
+
+    if (document.querySelectorAll(`.join-us`)) {
+      var elements = Array.from(document.querySelectorAll(`.join-us`));
```

Comment:
It turns out that `querySelectorAll` returns a static NodeList, so we don't need the `Array.from` anymore. We might want to file a separate issue to clean up `Array.from` everywhere, though, and add one here even though it's new code.

Reply:
I really wanna get this PR merged so let's tackle ^ as a follow-up: https://github.com/mozilla/foundation.mozilla.org/issues/3109